OpenRA.Mods.Common/Traits/Carryable.cs

```diff
@@ -15,38 +15,41 @@
 namespace OpenRA.Mods.Common.Traits
 {
 	[Desc("Can be carried by actors with the `Carryall` trait.")]
-	public class CarryableInfo : ITraitInfo
+	public class CarryableInfo : UpgradableTraitInfo
 	{
 		[UpgradeGrantedReference]
-		[Desc("The upgrades to grant to self while waiting or being carried.")]
-		public readonly string[] CarryableUpgrades = { };
+		[Desc("The condition to grant to self while a carryall has been reserved.")]
+		public readonly string ReservedCondition = "carryall-reserved";
```

Comment:
Does anyone have an opinion about granting these automatically (where it isn't obvious to modders where they are granted) vs defaulting to nothing and requiring the mods to define them (redundancy and trickier upgrade rules).

I'm leaning towards the second option, so adding the fixup requested tag until I have time to fix it.